apps/rd/rd-judicial-data-load/demo-01.yaml

```diff
@@ -8,3 +8,4 @@ spec:
       schedule: "15 12 * * *"
       environment:
         DELETE_ORG: true
+      image: hmctspublic.azurecr.io/rd/rd-judicial-data-load:pr-518
```

Comment:
you need to supply an image policy reference here, see example:
https://github.com/hmcts/cnp-flux-config/blob/e0be11b80c2225453b733ea18688a3b284ca0c0f/apps/rpe/draft-store-service/draft-store-service.yaml#L12

Reply:
updated the image full path